### DISCUSSION TITLE
Looking for cortexa35-poky-linux ARM support

### DISCUSSION THREAD

**Discussion by mdear** [AppImage/AppImageKit#1303] [Q&A]
Hello,

I'm cross-compiling on an x86_64 build VM for an i.MX8 cortexa-35-poky-linux platform.  Does this tool work in this environment ?

I am aware of another discussion https://github.com/orgs/AppImage/discussions/1294 that is similar, but my application must run quickly and efficiently on a series of production ARM boards (CrossControl v700/v1200) , whereas this discussion talks about x86_64 emulation on ARM platforms.

**Comment by TheAssassin** [comment]
Define "this tool", please.

**Reply by mdear** [reply]
As this is the AppImage discussions page, I am asking whether a variant of appimagetool-x86_64.AppImage exists that runs on an x86_64 platform and supports packaging for an i.MX8DualXPlus cortexa35-poky-linux architecture.

**Reply by TheAssassin** [reply]
There are more tools in the AppImage core project. That's why I was asking for more details.

I think it is more efficient to continue over at linuxdeploy's repository for now, as you will most likely use appimagetool via linuxdeploy's AppImage plugin anyway.

**Comment by TheAssassin** [comment]
Use case and discussion can be found in https://github.com/linuxdeploy/linuxdeploy/issues/258.

**Comment by johnnynunez** [comment]
https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/

**Reply by probonopd** [reply]
Currently only for paid GitHub plans unfortunately.
